### ISSUE TITLE
Step1中使用了目标域的标签比较准确率

### ISSUE THREAD

**Issue by HaydenYue** [phoenixdyf/Theory-guided-Progressive-Transfer-Learning-Network#2]
您好！
acc_test = accuracy_score(np.argmax(predict_prob_target.cpu().detach().numpy(),axis = -1),
                                      np.argmax(label_target.cpu().detach().numpy(),axis = -1))
训练过程中为什么能使用label_target来比较准确率保存模型。
按理只有测试阶段才能见到data1中所包含的label啊.
